public/locales/en-US/send.ftl

```diff
@@ -117,3 +117,146 @@ passwordIsSet = Password set
 maxPasswordLength = Maximum password length: { $length }
 # A short status message shown when there was an error setting the password
 passwordSetError = This password could not be set
+#
```

Comment:
What's up will these commented lines?

If they're not needed, please remove them. This is going to show up as one comment for the next valid string.

Reply:
I think danny is trying to differentiate between old strings, strings that are shared across old and new as well as new strings. I will remove the annotations. Cool if i leave a space?

Reply:
Looks like i can add FTL section heads

https://github.com/l20n/l20n-syntax-fixtures/blob/master/parser/ftl/04-sections.ftl

Reply:
Sorry, reference is here. That's a very old (and obsolete) project
https://projectfluent.org/fluent/guide/

Reply:
Per danny's note above, the commented lines are currently in prod. Since they are already localized, i think the GH diff does not matter here